website/source/docs/guides/connect-gateways.md

```diff
@@ -0,0 +1,348 @@
+                     -token=<token for the primary dc gateway>
+```
+
+Next, create a [centralized configuration](LINK) file for all the sidecar
+proxies in your primary datacenter called `proxy-defaults.hcl`. This file will
+instruct the sidecar proxies to send all their inter-datacenter through the
+gateway. It should contain the following:
+
+```
+Kind = "proxy-defaults"
+Name =  "global"
+MeshGateway = "local"
+```
+
+Write the centralized configuration you just created with the following command.
+
+```
+consul config write proxy-defaults.hcl
+```
+
+### Repeat the Process to Deploy the Gateway for your Secondary Datacenter
+
+Repeat the above process for the secondary datacenter. Register and start the
+gateway in your secondary datacenter with the following command.
+
+```
+consul connect envoy -mesh-gateway -register \
+                     -address "<your private address>" \
+                     -wan-address "<your externally accessible address>"\
+                     -token=<token for the secondary dc gateway>
+```
+
+Next, create a centralized configuration file for all the sidecar proxies in
+your secondary datacenter called `proxy-defaults.hcl`. This file will instruct
+the sidecar proxies to send all their inter-datacenter through the gateway. It
+should contain the following:
+
+```
+Kind = "proxy-defaults"
+Name =  "global"
+MeshGateway = "local"
+```
+
+Write the centralized configuration you just created with the following command.
+
+```
+consul config write proxy-defaults.hcl
+```
+
+Once this step is complete, you will have set up Consul Connect with gateways
+across multiple datacenters. Now you are ready to register the services that
+will use Connect.
+
+## Register a Service in Each Datacenter to Use Connect
+
+You can register a service to use a sidecar proxy by including a sidecar proxy
+stanza in its registration file. For this guide, you can use netcat to act as a
+backend service and register a dummy service called web to represent the client
+service. Those names are used in our examples. If you have services that you
+would like to connect, feel free to use those instead.
+
+~> **Caution:** Connect takes its default intention policy from Consul’s default
+ACL policy. If you have set your default ACL policy to deny (as is recommended
+for secure operation)  and are adding Connect to already registered services,
+those services will lose connection to each other until you set an intention
+between them to allow communication.
+
+### Register a back end service in one datacenter
+
+In one datacenter register a backend service and add an Envoy sidecar proxy
+registration. To do this you will either create a new registration file or edit
+an existing one to include a sidecar proxy stanza. If you are using netcat as
+your backend service, you will create a new file called `netcat.json` that would
+contain the below snippet. Since you have ACLs enabled, you will have to create
+a token for the service.
+
+```
+{
```

Comment:
I don't see where we tell them the netcat/socat command to run that listens on port 8181? Are we assuming they already know what to do from the first connect guide?